apps/dashboard/services/graphql/index.ts

```diff
@@ -1,26 +1,27 @@
-import { ApolloClient, HttpLink, InMemoryCache } from "@apollo/client"
-import { registerApolloClient } from "@apollo/experimental-nextjs-app-support/rsc"
+import { ApolloClient, HttpLink, InMemoryCache } from "@apollo/client";
+import { registerApolloClient } from "@apollo/experimental-nextjs-app-support/rsc";
 
-import { loadDevMessages, loadErrorMessages } from "@apollo/client/dev"
+import { loadDevMessages, loadErrorMessages } from "@apollo/client/dev";
 
-import { coreUrl } from "./config"
+import { coreUrl } from "./config";
 
 export const apollo = (token: string) =>
   registerApolloClient(() => {
     return new ApolloClient({
       cache: new InMemoryCache(),
       link: new HttpLink({
         headers: {
+          // authorization: `Bearer ${token}`,
```

Comment:
please remove or add comment

Reply:
yes only `Oauth2-Token` is needed here. authorization is for kratos but dashboard only relies on hydra token